File: platform/wlmedia_android/wlmedia/src/main/java/com/ywl5320/wlmedia/listener/WlOnBufferDataListener.java
package com.ywl5320.wlmedia.listener;

/**
 * author : ywl5320
 * e-mail : ywl5320@163.com
 * desc   : wlmedia
 * date   : 2024/12/14
 */
public interface WlOnBufferDataListener {
    long onBufferByteLength();

    byte[] onBufferByteData(long position, long bufferSize);
}

File: wlmedia_hm/libwlmedia/src/main/ets/com/ywl5320/wlmedia/listener/WlOnBufferDataListener.ts
/**
 * author : ywl5320
 * e-mail : ywl5320@163.com
 * desc   : wlmedia
 * date   : 2025/1/14
 */
export interface WlOnBufferDataListener {
  onBufferByteLength(): number;

  onBufferByteData(position: number, bufferSize: number): Uint8Array | null;
}